lib/restforce/collection.rb

```diff
@@ -31,6 +31,11 @@ def size
     end
     alias_method :length, :size
 
+    # Return whether the collection contains any objects.
```

Comment:
Good point. I'd missed that `#empty?` was specifically an `Array` method and not provided by `Enumerable`. I've quickly verified you can call `#any?` without passing a block and it should give you the inverse of `#empty?`. At least, assuming there's no scenario where we could get a collection that only contains `false` or `nil` values, which I can't recall ever running into.

I frequently find myself writing code to handle an empty query result, so I would find the syntax a little more expressive than doing `! #.any?` everywhere. But I can understand if you'd prefer to keep the interface minimal and stick to what `Enumerable` provides unless you've got a compelling reason to define additional methods (like for `#each`).